laravel-app/resources/views/profile/profile-form.blade.php

```diff
@@ -0,0 +1,114 @@
+	
+						@error('last_name')
+							<span class="invalid-feedback" role="alert">
+								<strong>{{ $message }}</strong>
+							</span>
+						@enderror
+					</div>
+				</div>
+	
+				<div class="row justify-content-center mb-1 mb-3">
+					<div class="col-md">
+						<label for="username">Username</label>
+						<input id="username" type="text" class="form-control @error('username') is-invalid @enderror" name="username" value="{{ $user->username }}" autocomplete="username" placeholder="Username" autofocus>
+	
+						@error('username')
+							<span class="invalid-feedback" role="alert">
+								<strong>{{ $message }}</strong>
+							</span>
+						@enderror
+					</div>
+				</div>
+
+				<div class="row justify-content-center mb-1 mb-3">
+					<div class="col-md">
+						<label for="email">Email</label>
+						<input id="email" type="text" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ $user->email }}" autocomplete="email" placeholder="Email" autofocus>
+	
+						@error('email')
+							<span class="invalid-feedback" role="alert">
+								<strong>{{ $message }}</strong>
+							</span>
+						@enderror
+					</div>
+				</div>
+
+				<div class="row justify-content-center mb-1 mb-3">
+					<div class="col-md">
+						<label for="bio">Bio</label>
+						<input id="bio" type="text" class="form-control @error('bio') is-invalid @enderror" name="bio" value="{{ $user->users_information->bio }}" autocomplete="bio" placeholder="Bio" autofocus>
+	
+						@error('bio')
+							<span class="invalid-feedback" role="alert">
+								<strong>{{ $message }}</strong>
+							</span>
+						@enderror
+					</div>
+				</div>
+
+				<div class="row justify-content-center mb-1 mb-3">
+					<div class="col-md">
+						<label for="gender">Gender</label>
+						<select id="gender" class="form-select @error('gender') is-invalid @enderror" name="gender" autocomplete="gender" autofocus>
+							<option value="" disabled>Select gender</option>
+							<option value="male" {{ old('gender', $user->users_information->gender) === 'male' ? 'selected' : '' }}>Male</option>
+							<option value="female" {{ old('gender', $user->users_information->gender) === 'female' ? 'selected' : '' }}>Female</option>
+							<option value="other" {{ old('gender', $user->users_information->gender) === 'other' ? 'selected' : '' }}>Other</option>
+						</select>
+						@error('gender')
+							<span class="invalid-feedback" role="alert">
+								<strong>{{ $message }}</strong>
+							</span>
+						@enderror
+					</div>
+				</div>
+	
+				<div class="row p-2 mb-0 mt-3">
+					<div class="col-md text-center">
+						<a href="{{ route('profile.index') }}" class="btn btn-secondary"> {{ __('Cancel') }} </a>
+					</div>
+					<div class="col-md text-center">
+						<button type="submit" class="btn btn-success">
+							{{ __('Update') }}
+						</button>
+					</div>
+				</div>
+
+				<div class="row p-2 mb-0 mt-3">
+				</div>
+	
+			</form>
```

Comment:
add new line

Reply:
added new line sir and fixed indention. Thank you ❤️